test/unit/util/dependencies/audit_test.py

```diff
@@ -43,6 +48,86 @@ def test_security_issue_entry(sample_vulnerability):
             == sample_vulnerability.security_issue_entry
         )
 
+    @staticmethod
+    @pytest.mark.parametrize(
+        "reference, expected",
+        [
+            pytest.param(
+                "CVE-2025-27516",
+                "https://nvd.nist.gov/vuln/detail/CVE-2025-27516",
+                id="CVE",
+            ),
+            pytest.param(
+                "CWE-611",
+                "https://cwe.mitre.org/data/definitions/611.html",
+                id="CWE",
+            ),
+            pytest.param(
+                "GHSA-cpwx-vrp4-4pq7",
+                "https://github.com/advisories/GHSA-cpwx-vrp4-4pq7",
+                id="GHSA",
+            ),
+            pytest.param(
+                "PYSEC-2025-9",
+                "https://github.com/pypa/advisory-database/blob/main/vulns/jinja2/PYSEC-2025-9.yaml",
+                id="PYSEC",
+            ),
+        ],
+    )
+    def test_reference_links(sample_vulnerability, reference: str, expected: list[str]):
+        result = Vulnerability(
+            name=sample_vulnerability.package_name,
+            version=sample_vulnerability.version,
+            id=reference,
+            aliases=[],
+            fix_versions=[sample_vulnerability.fix_version],
+            description=sample_vulnerability.description,
+        )
+
+        assert result.reference_links == (expected,)
+
+    @pytest.mark.parametrize(
+        "aliases,expected",
+        (
+            pytest.param(["A", "PYSEC", "CVE", "GHSA"], "CVE", id="CVE"),
+            pytest.param(["A", "PYSEC", "GHSA"], "GHSA", id="GHSA"),
+            pytest.param(["A", "PYSEC"], "PYSEC", id="PYSEC"),
+            pytest.param(["Z", "A"], "A", id="alphabetical_case"),
+        ),
+    )
+    def test_vulnerability_id(self, sample_vulnerability, aliases: list[str], expected):
+
+        result = Vulnerability(
+            name=sample_vulnerability.package_name,
+            version=sample_vulnerability.version,
+            id="DUMMY_IDENTIFIER",
+            aliases=aliases,
+            fix_versions=[sample_vulnerability.fix_version],
+            description=sample_vulnerability.description,
+        )
+
+        assert result.vulnerability_id == expected
+
+    def test_subsection_for_changelog_summary(self, sample_vulnerability):
```

Comment:
Reference is from PK:
https://github.com/exasol/databricks-virtual-schema/pull/54/files